firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/CheckForUpdateClientTest.java

```diff
@@ -311,4 +315,24 @@ public void isInstalledRelease_whenCodeHashesNotEqual_returnsFalse() {
     doReturn(TEST_CODEHASH_2).when(checkForUpdateClient).extractApkCodeHash(any());
     assertFalse(checkForUpdateClient.isInstalledRelease(TEST_RELEASE_NEWER_APK));
   }
+
+  @Test
+  public void extractApkCodeHash_ifKeyInCachedCodeHashes_doesNotRecalculateZipHash() {
+
+    try (MockedStatic mockedReleaseIdentificationUtils =
```

Comment:
we usually don't have a try block in tests. Does this throw an exception?

Reply:
Based it on this: https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#48 where they use a try resource block to make sure the mocking of the static method gets closed and is contained to that block

Reply:
ooo nice. TIL :)